docs/web-player.md

```diff
@@ -65,46 +62,49 @@ import '@liveryvideo/player';
 
 ### HTML
 
-?> Replace the Livery Demo stream id (`5ddb98f5e4b0937e6a4507f2`) below by your own.
-
 ```html
 <livery-player streamid="5ddb98f5e4b0937e6a4507f2"></livery-player>
 ```
 
+?> Replace the Livery Demo stream id (`5ddb98f5e4b0937e6a4507f2`) above by your own.
+
 Please see the [LiveryPlayer](#liveryplayer) API documentation below for additional attributes etc.
 
 ### CSS
 
-To fix the size of the livery-player element for the video etc. to be fit inside, use something like:
+The livery-player element is displayed as a `block`, e.g: using the full width available by default.
+
+If no height is specified it will use the video aspect ratio (default `16:9`) to determine the height intrinsically.
+
+Alternatively you can specify the size extrinsically, to prevent it from changing as the stream is loaded, e.g:
 
 ```css
 livery-player {
-  width: 100%;
   height: 50vh;
 }
 ```
 
-Use a specific extrinsic size to prevent that from changing intrinsically as the stream is loaded.
+See also: [MDN Sizing items in CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Sizing_items_in_CSS)
 
-E.g: Don't (just) use `min-height` but (also) specify `height` for the livery-player (directly or via its containing elements).
+When you use a fixed height like this the video contents of the player will be fit inside the available space based on the `Fit` mode configured in the Livery Portal for this stream (e.g: `CONTAIN` or `COVER` or ..).
```

Comment:
When you use a fixed height like this the video contents of the player will fit inside the available space based on the `Fit` mode configured in the Livery Portal for this stream (e.g: `CONTAIN` or `COVER` or ..).

Reply:
That doesn't seem correct to me 🤔  Using a fixed height will not make the contents fit by itself; it will cause the layout engine to to fit it. Perhaps it should be: "contents of the player will be fitted inside"?